Formula/png++.rb

```diff
@@ -12,9 +11,74 @@ class Pngxx < Formula
     sha256 "f2e242ee428f191645418a9897eb2fd729408dd67d04e7af4cefc7dcb5715250" => :mavericks
   end
 
+  head do
+    url "svn://svn.savannah.nongnu.org/pngpp/trunk"
+  end
+
   depends_on "libpng"
 
+  patch :DATA
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!